wiki/Skinning/Compendium/en.md

```diff
@@ -0,0 +1,261 @@
+
+## H
+
+- [Hakurei Reimu](https://osu.ppy.sh/forum/t/38951) by [MLGnom](https://osu.ppy.sh/u/46620)![][o!s]
+- [Hanami](https://osu.ppy.sh/forum/t/256243) by [Raidexx](https://osu.ppy.sh/u/2244606)![][o!s]![][o!t]![][o!c]
+- [Hanayome to Maou](https://osu.ppy.sh/forum/t/471178) by [atturbo555](https://osu.ppy.sh/u/5730417)![][o!s]
+- [Hatsune Miku Test](https://osu.ppy.sh/forum/t/180796) by [Elzapatoverde](https://osu.ppy.sh/u/3717733)![][o!s]![][o!c]
+- [Hestia](https://osu.ppy.sh/forum/t/460396) by [Heavenly](https://osu.ppy.sh/u/3300687)![][o!s]
+- [Hexarcle](https://osu.ppy.sh/forum/t/385047) by [BDWZ](https://osu.ppy.sh/u/5610209)![][o!s]![][o!c]![][o!m]
+- [Hikari Kyokusei](https://osu.ppy.sh/forum/t/515774) by [eltoroguaco](https://osu.ppy.sh/u/1439487)![][o!s]
+- [HueShift](https://osu.ppy.sh/forum/t/470410) by [JapanYoshiTheGamer](https://osu.ppy.sh/u/616511)![][o!s]![][o!t]![][o!c]![][o!m]
+
+## I
+
+- [IA - Aria on the Planetes](https://osu.ppy.sh/forum/t/239399) by [Elzapatoverde](https://osu.ppy.sh/u/3717733)![][o!s]![][o!c]
+- [IIDX22 Style](https://osu.ppy.sh/forum/t/426839) by [Toshiyuki Doma](https://osu.ppy.sh/u/792057)![][o!m]
+- [iMCrafter](https://osu.ppy.sh/forum/t/460563) by [iMCrafter](https://osu.ppy.sh/u/7993300)![][o!s]
+- [Initial D](https://osu.ppy.sh/forum/t/647964) by [FroxXyyy](https://osu.ppy.sh/u/9212147)![][o!s]
+- [Izayoi Sakuya](https://osu.ppy.sh/forum/t/32589) by [MLGnom](https://osu.ppy.sh/u/46620)![][o!s]
+
+## K
+
+- [Kagerou Project/Mekaku City Actors](https://osu.ppy.sh/forum/t/201016) by [Elzapatoverde](https://osu.ppy.sh/u/3717733)![][o!s]![][o!t]![][o!c]![][o!m]
+- [Kirisame Marisa](http://osu.ppy.sh/forum/t/24495) by [MLGnom](https://osu.ppy.sh/u/46620)![][o!s]
+- [Kochiya Sanae](https://osu.ppy.sh/forum/t/33140) by [MLGnom](https://osu.ppy.sh/u/46620)![][o!s]
+- [KRUEL](https://osu.ppy.sh/forum/t/502972) by [osu_Sora](https://osu.ppy.sh/u/7095266)![][o!s]![][o!c]![][o!m]
+- [Kurokami's skin](https://osu.ppy.sh/forum/t/134007) by [Kurokami](https://osu.ppy.sh/u/260933)![][o!s]![][o!t]![][o!c]![][o!m]
+
+## L
+
+- [Line Meet The Flat Skin](https://osu.ppy.sh/forum/t/483666) by [atturbo555](https://osu.ppy.sh/u/5730417)![][o!s]![][o!t]![][o!c]![][o!m]
+- [Lithe](https://osu.ppy.sh/forum/t/58842) by [Xiao](https://osu.ppy.sh/u/17894)![][o!s]
+- [Loli](https://osu.ppy.sh/forum/t/146503) by [HandHeldPillow](https://osu.ppy.sh/u/2706545)![][o!s]
+- [Lorem Ipsum](https://osu.ppy.sh/forum/t/349104) by [Elzapatoverde](https://osu.ppy.sh/u/3717733)![][o!s]![][o!c]
+- [LoveLive! School Idol Festival](https://osu.ppy.sh/forum/t/539048) by [Nitrous ](https://osu.ppy.sh/u/7293512)![][o!m]
+- [LoveLive! School Idol Festival](https://osu.ppy.sh/forum/t/586940) by [Nitrous](https://osu.ppy.sh/u/7293512)![][o!s]
+- [Love Live](https://osu.ppy.sh/forum/t/269765) by [RainAid ](https://osu.ppy.sh/u/3344403)![][o!s]![][o!t]![][o!c]
```

Comment:
some of these has an extra space after the username (this happens 7 times)

Reply:
Thats bad. That probably also happened on some skin names. Will try to fix that.